resources/models/home.py

```diff
@@ -358,7 +376,11 @@ def custom_serve(self, request, *args, **kwargs):
 
     project_info_block = ProjectInfoBlock.objects.all()
 
-    context = self.get_context(request)
+    path_components = kwargs.get('path_components', [])
+
+    list(map(lambda x: " ".join(x.split("-")), path_components))
```

Comment:
There's also a python [String replace()](https://www.tutorialspoint.com/python/string_replace.htm) which might be nicer